README.md

```diff
@@ -61,6 +61,31 @@ pub struct Rgb {
 }
 ```
 
+### Customizing single fields
+
+This can be particular handy if your structure uses a type that does not implement `Arbitrary` or you want to have more customization for particular fields.
+
+```rust
+#[derive(Arbitrary)]
+pub struct Rgb {
+    // set `r` to Default::default()
+    #[arbitrary(default)]
+    pub r: u8,
+
+    // set `g` to 255
+    #[arbitrary(value = "255")]
+    pub g: u8,
+
+    // generate `b` with a custom function
```

Comment:
And I think we should document the type of the expected function, something like

```suggestion
    // Generate `b` with a custom function of type
    //
    //     fn(&mut Unstructured) -> arbitrary::Result<T>
    //
    // where `T` is the field's type.
```

Reply:
Addressed in https://github.com/rust-fuzz/arbitrary/pull/129/commits/386de854122cbfab158bd99ffd7b7f4c46c8cfe9